src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -313,7 +313,8 @@
     <value>Rerun the command and pass --force to accept and create.</value>
   </data>
   <data name="RunHelpForInformationAboutAcceptedParameters" xml:space="preserve">
-    <value>Run dotnet {0} --help for usage information.</value>
+    <value>Run dotnet {0} --help for usage information.
+See https://aka.ms/dotnet-install-templates to learn how to install additional template packs.</value>

```

Comment:
I assume this link will resolve to something specific to installing templates. 
Right now it redirects to: https://docs.microsoft.com/en-us/aspnet/core/